snap-utils/usr/libexec/miriway-session-startup-snap

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -eu
+
+source "${SNAP}"/usr/libexec/miriway-session-startup "$@"
```

Comment:
@mattkae you commented that "Using source or exec made it not work though". So is this good now?

Reply:
Confused. Sourcing or executing shell script from another shell script doesn't work, but forking it, does…?

Sounds like systemd expects a fork somewhere?

Reply:
The reason is "source: not found".

But `.` works

Reply:
I was tricksed by `exec` not working :P